### ISSUE TITLE
[Adaptive Cards - iOS - Column.Size.Ratio.json]: Voice Over is conveying the same name for the all images under column size screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8425]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to ' Column.Size.Ratio.json' and activate it.
7. Card will appear.
8. Navigate to all images under the card and observe the issue.

**Actual Result:**
While navigating to the all images under the card, screen reader is conveying the same name for the all images. It is announcing as 'Cat image' for the all image.

**Expected Result:**
Screen Reader should convey the different and unique name for the all images under the card.

**Note:**
This issue is repro throughout the application for the same type of control.

**User Impact:**
It will impact the screen reader user if the same name is announced by the screen reader as user will not be able to differentiate between all images.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228770986-0d55e0d7-0575-4fd7-92db-2c2612cfca6e.mov

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
